src/components/pages/donation_confirmation/AddressKnown.vue

```diff
@@ -10,19 +10,19 @@
 
 		<div class="address-summary">
 			<p v-if="addressType === 'person'" v-html="$t( 'donation_confirmation_address_person', {
-				salutation: salutation,
+				salutation: this.salutation.display,
```

Comment:
You don't need `this.` here. Your IDE might be getting confused, it has a warning on mine too which I think has to do with the script beinf marked as ts. You can change this and the other template variables back.